playbooks/apache2.yaml

```
@@ -0,0 +1,10 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Setup apache2 server and configurations.
+---
+- name: Setup apache2 server and configurations
+  hosts: all
```

```suggestion
  hosts: apache_servers
```

Thoughts @NightTsarina? (Also for other playbooks below.)